static/main.js

```diff
@@ -67,13 +65,19 @@ function init() {
         try {
             katex.render(input.value, math, options);
         } catch (e) {
-            if (e.__proto__ == katex.ParseError.prototype) {
+            if (e.__proto__ === katex.ParseError.prototype) {
                 console.error(e);
             } else {
                 throw e;
             }
         }
     }
+
+    if (module.hot) {
+        module.hot.accept('../katex.webpack.js', reprocess);
+    }
 }
 
 init();
+
+export default katex;
```

Comment:
Now that we have separate entry points we should be able to get rid of this.

Reply:
@ylemkimon I guess we still need this given the reasons you gave in the main thread.

Reply:
@kevinbarabash Yes, in order to export KaTeX and make it accessible from console and other scripts.